packages/dependency-profiles/package.json

```diff
@@ -50,6 +50,7 @@
     "@fluentui-react-native/menu-button": "*",
     "@fluentui-react-native/merge-props": "*",
     "@fluentui-react-native/notification": "*",
+    "@fluentui-react-native/overflow": "*",
```

Comment:
While we're here let's add chip, divider, spinner, tablist and tooltip which seem to be missing